### PR TITLE
Small misc improvements

### PR DESCRIPTION
**Pull request by samerton**

### CODE REVIEW

File: core/classes/Core/URL.php [hunk 1]
```diff
@@ -85,6 +85,16 @@ private static function buildNonFriendly(string $url, string $params): string {
         return (defined('CONFIG_PATH') ? CONFIG_PATH : '') . '/index.php?route=' . $url . ((substr($url, -1) == '/') ? '' : '/');
     }
 
+    /**
+     * Build an asset path
+     *
+     * @param string $path Contains the asset path relative to the root Nameless directory
+     * @return string
+     */
+    public static function buildAssetPath(string $path): string {
```
Comment (tadhgboyle) [inline core/classes/Core/URL.php:94]: Reminder to self to do a bulk replacement and utilize this everywhere! Great addition

File: core/classes/Database/DB.php [hunk 2]
```diff
@@ -474,9 +474,9 @@ public function addColumn(string $table, string $column, string $attributes): bo
      * column, operator (default =), value, and glue (default AND).
      * @return array The where clause string, and parameters to bind.
      */
-    private function makeWhere(array $clauses): array {
+    public static function makeWhere(array $clauses): array {
```
Comment (tadhgboyle) [inline core/classes/Database/DB.php:477]: I admit I am still unsure how making this public helps modules at all - but won't consider this blocking.
Could you explain just so I can get a better understanding of how this method could be used by a module without going through the `where` method (which implicitly calls this already)?

**Review by tadhgboyle — APPROVED** [review]

File: core/classes/Database/DB.php (hunk 2 above)
Comment (samerton) [reply core/classes/Database/DB.php:477]: Here's an example: https://github.com/samerton/Nameless-Resources/blob/develop/upload/modules/Resources/classes/Resources.php#L121

It's to enable an easier way to pass parameters such as pagination and filters into methods which will perform some SQL, e.g. a repository class

Comment (partydragen) [reply core/classes/Database/DB.php:477]: SMART!!!! This ise usefull many places!

**Review by partydragen — APPROVED** [review]

**Pull request merged**